packages/r/reveal.js.json

```
@@ -16,14 +16,8 @@
       {
         "basePath": "",
         "files": [
-          "css/**/*.css",
-          "js/*.js",
-          "lib/**/*.js",
-          "lib/**/*.css",
-          "lib/**/*.eot",
-          "lib/**/*.ttf",
-          "lib/**/*.woff",
-          "plugin/**/*.js"
+          "dist/**",
```

```suggestion
          "dist/**/*.@(js|css|eot|ttf|woff)",
```
Open globstar is dangerous and leaves cdnjs open to abuse, let's specify the file types we need